### ISSUE TITLE
open existing files as empty

### ISSUE THREAD

**Issue by sandikata** [jimeh/.emacs.d#13]
After latest git pull, emacs has got a weird behavior, when i try to open a file, for example emacs archlinux.yml
it opens empty file, but not archlinux.yml which have content inside.

**Comment by sandikata** [comment]
https://i.imgur.com/bVbv1Mm.png

**Comment by jimeh** [comment]
Apologies, it's not that the file is empty, it's that as of 0.7.0 and the switch from persp-mode to tab-bar-mode/desktop.el, I set it to load a "default" desktop/session on startup, which leads it to closing/removing any files were passed to it emacs. Sorry I didn't think of that, my workflow is a bit different so it's not an issue I ran into myself.

I've fixed it in 6003c899945c50017be0f1c0cee9953a6c91e20b so it behaves like normal again, and desktop/session stuff is only started after selecting/creating a named desktop with `C-z C-z C-s`.
